rasa/utils/tensorflow/data_generator.py

```diff
@@ -3,15 +3,15 @@
 import logging
 import scipy.sparse
 import numpy as np
-import tensorflow as tf
+from tensorflow.python.keras.utils.data_utils import Sequence
```

Comment:
I thought we don't want to use `tf.python` as its private. Can't we leave it to `tf.keras` as that seems to be the public API from [TF 2.6 onwards](https://github.com/tensorflow/tensorflow/releases?page=2)

Reply:
That's needed because of the 
> for data generators, the keras export does not work well (the adapters ALL_ADAPTER_CLS in tf's data_adapter won't recognise it, leading to ValueError: Failed to find data adapter that can handle input: <class 'rasa.utils.tensorflow.data_generator.RasaBatchDataGenerator'>,)